src/hacbs/components/PipelineRunDetailsView/PipelineRunDetailsView.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { NamespaceContext } from '../../../components/NamespacedPage/NamespacedPage';
+import { PipelineRunGroupVersionKind } from '../../models';
+import { PipelineRunKind } from '../../types';
+import DetailsPage from '../ApplicationDetails/DetailsPage';
+import PipelineRunDetailsTab from './tabs/PipelineRunDetailsTab';
+import PipelineRunEnterpriseContractTab from './tabs/PipelineRunEnterpriseContractTab';
+import PipelineRunEventsTab from './tabs/PipelineRunEventsTab';
+import PipelineRunLogsTab from './tabs/PipelineRunLogsTab';
+import PipelineRunTaskRunsTab from './tabs/PipelineRunTaskRunsTab';
+import PipelineRunYamlTab from './tabs/PipelineRunYamlTab';
+
+type PipelineRunDetailsViewProps = {
+  pipelineRunName: string;
+};
+
+export const PipelineRunDetailsView: React.FC<PipelineRunDetailsViewProps> = ({
+  pipelineRunName,
+}) => {
+  const { namespace } = React.useContext(NamespaceContext);
+  const navigate = useNavigate();
+
+  const [pipelineRun, loaded] = useK8sWatchResource<PipelineRunKind>({
+    groupVersionKind: PipelineRunGroupVersionKind,
+    name: pipelineRunName,
+    namespace,
+  });
+
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+
+  return (
+    <>
+      <React.Fragment>
+        <DetailsPage
+          breadcrumbs={[
+            { path: '/app-studio/pipelineruns', name: 'Pipelineruns' },
+            {
+              path: `/app-studio/applications/:${pipelineRunName}`,
+              name: pipelineRunName,
+            },
+          ]}
```

Comment:
we should discuss this one - agreed that the existing breadcrumb needs to change, but the breadcrumb above is odd, that's not how a user will navigate to the pipeline details. they will most likely be on the list page and then click a pipelinerun. so using the breadcrumb as a navigational aid, they cannot get back to the pipelinerun list they were just on? and as a relational guide to where they are, it just seems convoluted because they did not visit those areas prior.

Reply:
as for the error you received, i was not able to reproduce it. i added a new component to my existing app, and then navigated to the pipelineruns list and did not get an error. however, load time was fairly slow so that might have to do with it.

Reply:
breadcrumbs aren't there to depict navigational history. There may be many ways to arrive at a details page all of which don't share the same navigational path.

Currently the breadcrumb goes to a standalone pipeline runs list. There is no standalone pipeline run list page in any of the designs anymore that I know of since the removal of the left nav item for "pipelines".

Reply:
i didn't say navigational history... it's more like where you are relationally and can trace your way back to where you were originally, hence the term breadcrumb

Reply:
For now, i've created a temp compromise for the details breadcrumb as `Applications > pipelinerun name` and we can work out the details in a future rev